xbmc/platform/win32/CharsetConverter.h

```diff
@@ -22,6 +22,7 @@
 
 #include <string>
 
+#if defined(TARGET_WINDOWS) || defined(TARGET_WIN10)
```

Comment:
I want to avoid having ifdefs in every file that includes this file so I chose to stick them in here to make calling code a bit cleaner